pkg/cloudevents/types/value.go

```diff
@@ -1,5 +1,7 @@
 package types
 
+// FIXME(alanconway) naming: URL, Time?
```

Comment:
I like sticking with the native type names as you have it.

Reply:
+1